SmallLibrary/index.html

```diff
@@ -0,0 +1,66 @@
+                <button class="button form__button_null" onclick="Main.insertNull(), Main.display('.array-display__list_source', Main.getArray());">null</button>
+                <button class="button form__button_undefined" onclick="Main.insertUndefined(), Main.display('.array-display__list_source', Main.getArray());">undefined</button>
+            </div>
+        </header>
+        <div class="content">
+            <div class="array-display">
+                <h3 class="array-display__header">All Elements of the Array:</h3>
+                <ul class="array-display__list_source"></ul>
+            </div>
+            <div class="operation-field">
+                <div class="for-each-block" onclick="Main.display('array-display__list_proc', Main.onForEachButtonClick());">
+                    <button class="button">forEach*</button>
+                    <label class="params">x * 2</label>
+                </div>
+                <div class="where-block" onclick="Main.display('.array-display__list_proc', Main.onWhereButtonClick());">
+                    <button class="button">where</button>
+                    <label class="params">10 &le; x &le; 20</label>
+                </div>
+                <div class="first-block">
+                    <button class="button" onclick="Main.display('.array-display__list_proc', Main.onFirstButtonClick());">first</button>
+                </div>
+                <div class="last-block">
+                    <button class="button" onclick="Main.display('.array-display__list_proc', Main.onLastButtonClick());">last</button>
+                </div>
+                <div class="select-block">
+                    <button class="button" onclick="Main.display('.array-display__list_proc', Main.onSelectButtonClick());">select</button>
+                    <label class="params">length</label>
+                </div>
+                <div class="skip-block">
+                    <button class="button" onclick="Main.display('.array-display__list_proc', Main.onSkipButtonClick());">skip</button>
+                    <input class="params" id="skip-block__number-input" type="text"/>
+                </div>
+                <div class="take-block">
+                    <button class="button" onclick="Main.display('.array-display__list_proc', Main.onTakeButtonClick());">take</button>
+                    <input class="params" id="take-block__number-input" type="text"/>
+                </div>
+            </div>
+            <div class="array-display">
+                <h3 class="array-display__header">Elements of the processed Array:</h3>
+                <ul class="array-display__list_proc"></ul>
+            </div>
+        </div>
+        <footer><span>*ForEach function is a void one that's why use console to see it's output</span></footer>
+        <script src="./small-lib/type-check-functions.js"></script>
+        <script src="./small-lib/array-functions.js"></script>
```

Comment:
In common case, use CamelCase notation (https://ru.wikipedia.org/wiki/CamelCase) for the naming of javascript files.
In you case,  the name should be arrayFunctions.js